cmd/dfuseeos/cli/init.go

```diff
@@ -53,19 +53,17 @@ func dfuseInitE(cmd *cobra.Command, args []string) (err error) {
 
 	if runProducer {
 		userLog.Printf("")
-		userLog.Printf("Here the key pair controlling 'eosio' to interact with your local chain:")
+		userLog.Printf("Here are the key pair controlling 'eosio' to interact with your local chain:")
```

Comment:
Here is the keypair that controls `eosio`. Use to interact with your local chain

Reply:
In my defense, it was late. Re-reading it, it makes no sense to use `are`. I would use Josh's comment with a small tweak:

`Here is the keypair that controls 'eosio'. Use it to interact with your local chain.`
or `Here is the keypair that controls 'eosio'. Use the keys to interact with your local chain.`